src/s3.js

```diff
@@ -78,9 +79,46 @@ const moveObjects = async (objects, oldPrefix, newPrefix) => {
   console.log(`Reset completed. ${newPrefix} folder is now repopulated`);
 };
 
+const uploadObject = async (fileName, file) => {
```

Comment:
`fileName` is a good clear name, but `file` is ambiguous here. Let's rename `file` -> `filePath`